lib/spack/docs/example_files/test.yaml

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
--- means: is this document part of a single stream (or not): https://en.wikipedia.org/wiki/YAML